src/blocks/ImageTextBlock.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { MaxWidthContainerComponent } from '../components/blockHelpers'
+import {
+  BaseBlockProps,
+  MarkdownHtmlComponent,
+  NativeColorPickerComponent,
+} from './BaseBlockProps'
+
+import {
+  FilledButtonComponent,
+  OutlinedButtonComponent,
+} from '../components/Buttons'
+
+const FilledButtonWithMarginComponent = styled(FilledButtonComponent)({
+  marginTop: '47px',
+})
+
+const OutlinedButtonWithMarginComponent = styled(OutlinedButtonComponent)({
+  marginTop: '47px',
+})
+
+const SectionComponent = styled('section')(
```

Comment:
i think we should reuse https://github.com/HedvigInsurance/web-next/blob/master/src/components/blockHelpers.tsx#L3 rather than creating new ones for each block

Reply:
also imo we should merge https://github.com/HedvigInsurance/web-next/pull/22 and reuse that. Thoughts?

Reply:
Good points, will do